src/test/java/life/qbic/omero/BasicOMEROClientSpec.groovy

```diff
@@ -11,6 +12,13 @@ import spock.lang.Specification
  * @since <versiontag>
  */
 class BasicOMEROClientSpec extends Specification {
+
+  def "first test"() {
+    expect:
+    1 + 1 == 2
+  }
+
```

Comment:
I would prefer to have a test like
composeImageDetailAddress(long imageId) == "actualLink"

Reply:
No can do with the current setup. Major refactoring would be required.